torchvision/csrc/cpu/video/Video.cpp

```diff
@@ -0,0 +1,393 @@
+
+#include "Video.h"
+#include <c10/util/Logging.h>
+#include <torch/script.h>
+#include "defs.h"
+#include "memory_buffer.h"
+#include "sync_decoder.h"
+
+using namespace std;
+using namespace ffmpeg;
+
+// If we are in a Windows environment, we need to define
+// initialization functions for the _custom_ops extension
+// #ifdef _WIN32
+// #if PY_MAJOR_VERSION < 3
+// PyMODINIT_FUNC init_video_reader(void) {
+//   // No need to do anything.
+//   return NULL;
+// }
+// #else
+// PyMODINIT_FUNC PyInit_video_reader(void) {
+//   // No need to do anything.
+//   return NULL;
+// }
+// #endif
+// #endif
+
+const size_t decoderTimeoutMs = 600000;
+const AVSampleFormat defaultAudioSampleFormat = AV_SAMPLE_FMT_FLT;
+// A jitter can be added to the end of the range to avoid conversion/rounding
+// error, small value 100us won't be enough to select the next frame, but enough
+// to compensate rounding error due to the multiple conversions.
+const size_t timeBaseJitterUs = 100;
```

Comment:
nit: this is not used

Reply:
Good point - will remove in the subsequent commit